LTHPasscodeViewController/LTHPasscodeViewController.m

```diff
@@ -49,7 +49,11 @@
  Any help would be greatly appreciated.
  */
 
+#if defined(LTH_APP_EXTENSIONS)
```

Comment:
I haven't used app extensions until now, so I need to ask how will this affect the above scenario? Can one define `LTH_APP_EXTENSIONS` just for the extension, so that the main app continues to use `windows[0]`?